sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1012,6 +1022,8 @@ pi_result piextPlatformCreateWithNativeHandle(pi_native_handle NativeHandle,
   return PI_INVALID_VALUE;
 }
 
+_pi_platform::~_pi_platform() { PiDevicesCache.clear(); }
```

Comment:
This seem redundant because PiDevicesCache is a a field of _pi_platform object:
    std::vector<std::unique_ptr<_pi_device>> PiDevicesCache;
and thus is destroyed when the _pi_platform object is destroyed. During the destruction of the PiDevicesCache vector each element is also destructed.

Reply:
Fixed. Thanks.